__tests__/nc_news.test.js

```diff
@@ -64,4 +82,45 @@ describe("nc_news",()=>{
             })
         })
     })
+    describe("GET /api/articles", ()=>{
+        test("Status: 200 returns array of article objects with its properties sorted by date desc and not include body property", ()=>{
+            return request(app)
+            .get("/api/articles")
+            .expect(200)
+            .then((response)=>{
+                const arrayOfArticles = response.body
+                arrayOfArticles.forEach((article)=>{
+                    expect(article).toMatchObject({
+                        article_id: expect.any(Number),
+                        title: expect.any(String),
+                        topic: expect.any(String),
+                        author: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number),
+                        article_img_url: expect.any(String),
+                        comment_count: expect.any(String)
+                    })
+                })
+            })
+        })
+        test("Array of objects is sorted by created_at in descending order", ()=>{
+            return request(app)
+            .get("/api/articles")
+            .expect(200)
+            .then((response)=>{
+                const arrayOfArticles = response.body
+                expect(arrayOfArticles).toBeSortedBy("created_at", {
+                    descending: true,
+                })
+            })
+        })
+        test("Status:404 when passing an invalid endpoint for api/articles", ()=>{
```

Comment:
Totally a valid test, just shouldn't be in with this describe block as it pertains to all endpoints, not just this one.